Formula/watchman.rb

```diff
@@ -1,28 +1,11 @@
 class Watchman < Formula
   desc "Watch files and take action when they change"
   homepage "https://github.com/facebook/watchman"
+  url "https://github.com/facebook/watchman/archive/v2021.05.31.00.tar.gz"
```

Comment:
They seem to have switched their version scheme to using dates about a year ago.